alcove/config.py

```
@@ -0,0 +1,308 @@
+            config_value=values.get("excerpt_chars"),
+            minimum=1,
+        ),
+        private_mode=_resolve_bool(
+            env_name="ALCOVE_PRIVATE",
+            config_value=values.get("private_mode"),
+            default=True,
+        ),
+    )
+
+
+def set_private_mode(enabled: bool) -> None:
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)